ui-mockups/src/article.html

```diff
@@ -171,8 +171,28 @@ <h1 class="card-title">Article title: My journey into Andela</h1>
                             because it is pain..." "There is no one who loves pain itself, who seeks after it and wants
                             to have it, simply because it is pain..."
                         </div>
-
                     </div>
+                    <!-- Comment section -->
+                    <div class="comments row ml-2 border-top border-bottom Z no-gutters">
+                            <h4 class="w-100">Comments (5)</h4>
+                            <p>Start a discussion not a fire. Post with kindness</p>
+                            <textarea rows="2" cols="100" name="comment" form="usrform">Post a comment</textarea>
+                        <div class="col col-md-1">
+                            <!-- Usr icon -->
+                            <div class="thumbnail">
+                                    <i class="fas fa-user-circle"></i>
+                            </div>
+                        </div>
+                        <div class="col col-md-11">
+                            <div class="panel panel-default">
```

Comment:
Are we using Bootstrap 3 here? We don't have `panel` and `thumbnail` in Bootstrap 4 but instead use `cards`.

Reply:
I agree with you @pngg

Reply:
@pngg The changes have been effected.